README.md

```diff
@@ -201,6 +202,26 @@ To enable teslausb to act as a wifi access point with the given SSID and passwor
    #export AP_IP='192.168.66.1'
    ```
 
+## Security
+
+As a little descussion of security for your Pi, please keep in mind the following items.
+
+1. If WiFi Access Point is configured, the AP password needs to be worth while. Make it something better than Passw0rd, more than 8 characters (odd number of upper/lower characters, numbers are the best.) 
+
+2. Change the passwords for the root and pi accounts. To do that, make the system RW and use the passwd command to change the two passwords. To do that, follow the instructions below and reboot your Pi: (Starting at the pi sign-on)
+
+```
+   ssh pi@teslausb.local
+   sudo -i
+   cd bin
+   ./remountfs_rw
+   passwd pi
+   passwd
```

Comment:
This will set the root password, meaning you could then log in as root with that password, which I think is less safe than the default "root has no valid password that you can log in with".

Reply:
Its interesting that the Pi has no valid password for root. That is different than the Linux systems I have managed. If that is truly the case, we can remove the passwd command that changes the root password. I just expected that it needs changing for sure. Its usually the first thing to change.

Reply:
You want to reject the request and I will submit it again without changing the root password?